source/website/src/views/Step3.vue

```diff
@@ -366,12 +367,12 @@ SPDX-License-Identifier: Apache-2.0
               return
             }
 
-            let valid_keys = ["name", "description", "data_type", "column_type", "pii_type", "nullable"]
+            let valid_keys = ["name", "description", "data_type", "dataType", "columnType", "column_type", "pii_type", "nullable", "externalUserIdType", "isMainEventTime"]
```

Comment:
Why this change? It does not appear to relate to liveramp.

Reply:
Resolved